src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -865,7 +866,7 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 		return this.groupView.getIndexOfEditor(editor);
 	}
 
-	private lastSelectedEditor: EditorInput | undefined;
+	private lastCtrlClickSelectedEditor: EditorInput | undefined;
```

Comment:
`CtrlClick` is a weird name, esp. since its `Command` on macOS, I would name this differently, keyboard agnostic.